rust/openvasd/src/config.rs

```diff
@@ -422,7 +422,8 @@ impl Config {
                 clap::Arg::new("enable-get-scans")
                     .env("ENABLE_GET_SCANS")
                     .long("enable-get-scans")
-                    .action(ArgAction::SetTrue)
+                    .num_args(0..=1)
```

Comment:
why? what was wrong with `--enable-get-scans` enables `\scans` without having to add '1, true, ...' as additional values?